File: services/telegramservice/DTOs/DTO/StartSessionModalDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DTOs.DTO
{
    public class StartSessionModalDTO
    {
        public required string ApiId { get; set; }
        public required string Hash { get; set; }
        public required string PhoneNumber { get; set; }
        public string? WithQR { get; set; }
    }
}

File: services/prod/src/main/java/com/torin/prod/dto/StartSessionModalDto.java
package com.torin.prod.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class StartSessionModalDto {
    private String apiId;
    private String hash;
    private String phoneNumber;
    private String withQR;
}
